ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,52 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+
+### Segment Routing Summary
+
+- SRTE is enabled.
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+
+### SRTE Policies
+
+| Endpoint | Color | Preference | Name | Description | Label Stack | Index  | Weight | Explicit Null |
+| -------- | ----- | ---------- | ---- | ----------- | ----------- | ------ | ------ | ------------- |
+{%         if router_traffic_engineering.segment_routing.policy_endpoints is arista.avd.defined %}
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoints | arista.avd.natural_sort %}
+{%                 for color in endpoint.colors | arista.avd.natural_sort %}
+{%                     set name = color.name | arista.avd.default('-') %}
+{%                     set description = color.description | arista.avd.default('-') %}
+{%                     for pathgroup in color.path_group | arista.avd.natural_sort %}
+{%                         set pathgroup_preference = pathgroup.preference | arista.avd.default('-') %}
+{%                         if pathgroup.explicit_null is arista.avd.defined('ipv4') %}
+{%                             set expnull = "ipv4" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null is arista.avd.defined('ipv6') %}
+{%                             set expnull = "ipv6" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null is arista.avd.defined('ipv4 ipv6') %}
+{%                             set expnull = "ipv4 ipv6" %}
+{%                         endif %}
+{%                         set expnull = expnull | arista.avd.default('-') %}
```

Comment:
You can delete the lines 25-33. As we don't have any requirement to check the values in that variable. 
(and)
Modify line 34: 
```
set expnull = pathgroup.explicit_null | arista.avd.default('-')
```

Reply:
The current scenario is printing out **-** if **expnull = none**